<ui version="4.0" >
 <class>RunSnipetDialog</class>
 <widget class="QDialog" name="RunSnipetDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Use the template</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Description</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="m_labelName" >
        <property name="text" >
         <string>&lt;b>This is the name&lt;/b></string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_descriptionLabel" >
        <property name="text" >
         <string>This is a long description of the template. This description explain how to use, or why, and all the possible parameter.</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_paramGroupBox" >
     <property name="title" >
      <string>&amp;Parameter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../application.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RunSnipetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>268</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel" >
     <x>270</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RunSnipetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>349</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel" >
     <x>359</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
